src/app/pages/send/send-crypto-asset-form/send-crypto-asset-form.routes.tsx

```diff
@@ -69,16 +69,20 @@ export const sendCryptoAssetFormRoutes = (
       <Route path={RouteUrls.SendBrc20SendForm} element={<Brc20SendForm />} />
       <Route path={RouteUrls.SendBrc20ChooseFee} element={<BrcChooseFee />} />
       <Route path={RouteUrls.SendBrc20Confirmation} element={<Brc20SendFormConfirmation />} />
-      <Route path={RouteUrls.SentBrc20Summary} element={<Brc20SentSummary />}></Route>
+      <Route path={RouteUrls.SentBrc20Summary} element={<Brc20SentSummary />} />
     </Route>
 
     <Route path={RouteUrls.SendCryptoAssetForm.replace(':symbol', 'stx')} element={<StxSendForm />}>
       {broadcastErrorDrawerRoute}
       {editNonceDrawerRoute}
-      {ledgerStacksTxSigningRoutes}
       {recipientAccountsDrawerRoute}
     </Route>
-    <Route path="/send/stx/confirm" element={<StacksSendFormConfirmation />} />
+    <Route
+      path={`${RouteUrls.SendCryptoAssetForm.replace(':symbol', 'stx')}/confirm`}
```

Comment:
This is working but seems a bit odd, doing a `replace(` in route param

Reply:
Open to other suggestions how to tidy these routes, not super clean atm. But we need to have a hierarchy to catch `btc` & `stx` before `:symbol`